dev/jenkins/run_docker.sh

```diff
@@ -38,6 +38,13 @@ function main {
     run_args+=" -e ALLUXIO_GIT_CLEAN=true"
   fi
 
+  if [ -n "${ALLUXIO_SONAR_ARGS}" ]
+  then
+    # write out to a file, in case there are spaces in the args
+    echo "ALLUXIO_SONAR_ARGS=${ALLUXIO_SONAR_ARGS}" > /tmp/.env
```

Comment:
you probably want this?

```
     echo "ALLUXIO_SONAR_ARGS=\"${ALLUXIO_SONAR_ARGS}\"" > /tmp/.env
```

Reply:
That doesn't seem to work. The quotes get included, which is not what I want. I think Docker already takes care if there are spaces in the values.